src/kp2bw/bw_import.py

```
@@ -0,0 +1,109 @@
+        "encrypted": False,
+        "folders": folders,
+        "items": items,
+    }
+
+
+def run_import(filepath: Path) -> None:
+    """Execute ``bw import bitwardenjson <filepath>`` as a subprocess."""
+    cmd = f'bw import bitwardenjson "{filepath}"'
+    logger.debug("Running bw import")
+    try:
+        output = check_output(cmd, stderr=STDOUT, shell=True)
```

Ruff (0.15.1)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)